opensearch/services/datasets/datasets.php

```diff
@@ -72,6 +72,22 @@ private function getDCOAuthorsByDataset($dataset) {
 		return $this->sparqlSelect($query);
 	}
 
+	/*
```

Comment:
If we want to support multiple data types for each dataset entry, then yes.  I am not sure on the performance hit - we'll just have to try it in test.